src/dft/backends/portfft/CMakeLists.txt

```diff
@@ -36,7 +36,11 @@ if (IS_DPCPP AND UNIX AND NOT FOUND_TARGETS)
   set(TARGETS_LINK_OPTIONS -fsycl-unnamed-lambda)
 
   # spir64 must be last in the list due to a bug in dpcpp 2024.0.0
-  set(TARGETS_TRIPLES "nvptx64-nvidia-cuda,spir64")
```

Comment:
IMO it would be a bit more consistent if `TARGETS_TRIPLES` is first set to `spir64` and then we prepend `nvptx64-nvidia-cuda` if possible, similarly to how we handle AMD.

Reply:
I agree. Done in [78a3eb5](https://github.com/oneapi-src/oneMKL/pull/511/commits/78a3eb551e7dabc1f49ec62bfb055bb008b6fbde).